mirrord-config/src/pod.rs

```diff
@@ -29,9 +29,9 @@ mod tests {
     #[rstest]
     fn default(
         #[values((Some("pod"), Some("pod".to_string())))] name: (Option<&str>, Option<String>),
-        #[values((None, "default"), (Some("namespace"), "namespace"))] namespace: (
+        #[values((None, None), (Some("namespace"), Some("namespace".to_string())))] namespace: (
             Option<&str>,
-            &str,
+            Option<String>,
```

Comment:
Nitpick but you can use `Option<&str>`, remove the `.to_string()` and change the assertion to `assert_eq!(outgoing.namespace.as_deref(), namespace.1);`

Reply:
lord of nitpicks <3

Reply:
leaving it as it's just a test..